pyalgostrategypool/vwap_crossover_v2.py

```diff
@@ -0,0 +1,61 @@
+"""
+    checkout:
+        - strategy specific docs here : https://algobulls.github.io/pyalgotrading/strategies/vwap_crossover/
+        - generalised docs in detail here : https://algobulls.github.io/pyalgotrading/strategies/common_regular_strategy/
+"""
+
+
+class VWAPCrossover(StrategyBase):
+    name = 'VWAP Crossover v2'
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.main_order_map = None
+
+    def initialize(self):
+        self.main_order_map = {}
+
+    def get_crossover_value(self, instrument):
+        hist_data = self.get_historical_data(instrument)
+        vwap = VWAP(hist_data)
+        crossover_value = self.utils.crossover(hist_data['close'], vwap)
+        return crossover_value
+
+    def strategy_select_instruments_for_entry(self, candle, instruments_bucket):
+        selected_instruments, meta = [], []
```

Comment:
`selected_instruments, meta = [], []`

leave a line after this

you have kept a blank line after this everywhere, lets do that here too

Reply:
done

Reply:
done